src/domains/news/components/NewsOptions/NewsOptions.tsx

```diff
@@ -16,10 +17,14 @@ export const NewsOptions = ({ categories, selectedAssets }: Props) => {
 	const { t } = useTranslation();
 
 	return (
-		<div className="p-8 border-2 rounded-lg border-theme-primary-contrast" data-testid="NewsOptions">
+		<div
+			className="p-8 border-2 rounded-lg bg-theme-background border-theme-primary-contrast"
+			data-testid="NewsOptions"
+		>
 			<div className="flex flex-col space-y-10">
-				<div className="flex justify-end">
-					<HeaderSearchBar />
+				<div className="flex items-center px-4 py-6 rounded shadow-xl">
+					<Input className="border-none shadow-none" placeholder="Search" />
```

Comment:
Hey @brenopolanski I think if you add a `padding: 1rem 0.75rem` as a styled css (py-4 and px-3 will not override the existing styling) will fix the glitching issue @faustbrian mentioned.

Reply:
Thanks @goga-m i solved using a styled wrapper :+1: